playbooks/collect_instance_statistics/query_elasticsearch.py

```diff
@@ -45,14 +45,52 @@ def valid_date(s):
         help='The last day on which statistics should be gathered.',
         required=True
     )
+    parser.add_argument(
+        '--host',
+        default='localhost',
```

Comment:
@mtyaka, since the host is configurable here, and we now have SSL and proper authentication, can't we reconfigure Elasticsearch to make its API available on the public IP address of the server? If the ELK VM and the Ocim prod VM are in the same OVH account and region, we can even restrict the access via a firewall rule.

CC @giovannicimolin @toxinu as the devops specialists.

Reply:
Yes, if Elasticsearch was accessible directly from the Ocim VM, this script would be much cleaner.

I didn't feel confident doing that in this ticket since I don't know much about our Elasticsearch setup and I'm not sure what security implications opening access to it would have, but if we can restrict access via firewall, and if @giovannicimolin and @toxinu agree, I can create a follow-up ticket to do that.